store/copr/store.go

```diff
@@ -109,6 +110,7 @@ func (s *Store) GetClient() kv.Client {
 	return &CopClient{
 		store:           s,
 		replicaReadSeed: s.nextReplicaReadSeed(),
+		numcpu:          runtime.GOMAXPROCS(0),
```

Comment:
Could it cause contention on the scheduler mutex?

Reply:
Yeah, moved this call to `Store`